shared/src/dateHelpers.ts

```diff
@@ -11,7 +11,7 @@ export const getCurrentDate = (): Dayjs => {
 };
 
 export const getJanOfCurrentYear = (): Dayjs => {
-  return dayjs().month(0);
+  return dayjs().month(0).date(1);
```

Comment:
Why is this addition necessary?

Reply:
We added it b/c Faraz was wondering about specifically-dated events. I have updated it to return dayjs().startOf('year') so that it resets to January 1st of the specific year.

**Source:** https://day.js.org/docs/en/manipulate/start-of#docsNav